core/src/main/java/io/lionweb/lioncore/java/model/ClassifierInstanceUtils.java

```diff
@@ -252,4 +250,40 @@ public static void addChild(
     Containment containment = _this.getClassifier().getContainmentByName(containmentName);
     _this.addChild(containment, child);
   }
+
+  public static ReferenceValue referenceTo(@Nonnull LanguageEntity<?> _this) {
+    // Unfortunately we cannot refer to LionCore and LionCoreBuiltins as this method get called
+    // during their initialization
+    if (_this.getLanguage() != null
+        && "LionCore_M3".equals(_this.getLanguage().getName())
+        && _this.getLionWebVersion() == LionWebVersion.v2024_1) {
+      return new ReferenceValue(_this, "LionWeb.LionCore_M3." + _this.getName());
```

Comment:
It feels like these magic strings appear several times over the PR. Maybe consolidate them?

Reply:
I will save these constants in a new class, called Autoresolve